chibuezeayogu/React-Redux
# #161155211 Implement view all courses

**What does the PR do?**
- This PR implements view all courses

**Description of Task to be completed?**
- ensure users can view all courses
- create app store
- create courses and authors mock API
- define app routes with react router dom
- write test
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-implement-view-course-161155211`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161155211](https://www.pivotaltracker.com/story/show/161155211)

**Screenshots**
- N/A
